.flake8

```diff
@@ -0,0 +1,13 @@
+[flake8]
+
+max-line-length=100
```

Comment:
Better `ignore = E501` Then only pylint will check length of string

Reply:
fixed